soundbay/data.py

```diff
@@ -288,6 +290,25 @@ def __call__(self, sample):
         return (sample - sample.min()) / (sample.max() - sample.min())
 
 
+class MinFreqSpectrogram:
+    """Cut the spectrogram frequency axis to make it start from min_freq"""
+    def __init__(self, min_freq_spectrogram, sample_rate):
+        self.min_freq_spectrogram = min_freq_spectrogram
+        self.sample_rate = sample_rate
+
+    def edit_spectrogram_axis(self, sample):
+        max_freq_in_spectrogram = self.sample_rate / 2
```

Comment:
maybe add a comment for using the max_freq parameter if it will be implemented in the future

Reply:
as of current implementation, inference is not affected in any way. I can implement the filtering for inference as well, but I think that deserves a seperate implementation that can be controlled through the relevant conf files (in parallel to the training conf files and implementation)

given the current implementation (and assuming no major changes), the inference will act just like normal, using spectrograms from freq=0 to sample_rate/2, and the output will depend solely on the trained model's ability to detect the signal, wherever it is within the spectrum

*about the comment, i've added a note within data.py, inside class implementation, and also added checks to make sure that  0 < min_freq_filtering < sample_rate/2
if/when we implement a max_freq filter similar to the min_freq one, id suggest we implement a check that takes place in both of the proccesses to make sure there are no contradictions